src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala

```diff
@@ -168,6 +170,19 @@ object ProjectFortis extends App {
               "consumerGroup" -> "$Default"
             )
           )
+        ),
+        "customevents" -> List(
+          ConnectorConfig(
+            "CustomEvents",
+            Map (
+              "policyName" -> envOrElse("CUSTOMEVENTS_EH_POLICY_NAME", "project-fortis-spark"),
```

Comment:
Oh wow, the nodejs EventHubs library [only supports the connection-string](https://github.com/Azure/azure-event-hubs-node/blob/1084724891a92ef6d1a5f1732a52d24ce871dee9/send_receive/lib/config.d.ts#L7-L8). Looks like we'll have to provide both unfortunately.